src/pages/_app.jsx

```diff
@@ -124,18 +129,28 @@ const WrappedApp = ({ Component, pageProps }) => {
 
     // Otherwise, load the page inside the content wrapper.
     return (
-      <AppRouteProvider>
-        <ContentWrapper
-          routeExperimentId={experimentId}
-          experimentData={experimentData}
-        >
-          <Component
-            experimentId={experimentId}
+      <ErrorBoundary
+        FallbackComponent={Error}
+        onError={(error, info) => {
+          // Only log errors to Slack if in production.
+          if (process.env.NODE_ENV !== 'production') return;
```

Comment:
could you put this logic into `postErrorToSlack`? I think putting it here makes it harder to read the `app.jsx` file

Reply:
moved